SDWebImage/SDWebImageCodersManager.h

```diff
@@ -10,13 +10,42 @@
 #import <Foundation/Foundation.h>
 #import "SDWebImageCoder.h"
 
-@interface SDWebImageCodersManager : NSObject<SDWebImageProgressiveCoder>
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I think we are deliberately not using `NS_ASSUME_NONNULL_BEGIN` in the project.

Reply:
Well. Remove that and add each nullability manually...

Reply:
Well in #1581 quote:

>Decided to make explicit nullable and nonnull and not use NS_ASSUME_NONNULL_BEGIN so everything is clearer when read. Done so in the implementation files as well